nabs/tests/test_plans.py

```diff
@@ -388,3 +388,69 @@ def test_daq_fixed_target_multi_scan(RE, daq, hw, sample_file):
     assert len(reads) == 24
     RE(msgs)
     summarize_plan(msgs)
+
+
+def orange(start, stop, num):
```

Comment:
This is such a nice utility function; why not use it for the scan?

Reply:
My thought was that I don't need to calculate `orange` for the step number (integer) case, since it can default to the normal `bp.scan` plan.  I could see it being useful to scientists though, maybe.